src/containers/Forms/index.tsx

```diff
@@ -4,13 +4,119 @@ import Container from '@material-ui/core/Container';
 import '../../App.css';
 import Login from '../../components/Login';
 import SignUp from '../../components/SignUp';
+import axios from 'axios';
 
+import {
+    Input,
+    Row,
+    Col
+} from 'antd';
```

Comment:
Looks like this is the only place that you are using `antd`, but there are Material-UI and/or React-Bootstrap components that already serve same/similar purpose. Can we try possibly re-use those instead of installing antd, which is more like a base design package? Just so that we keep our design consistent.

Reply:
It deleted using antd and put most logic into a component "MovieSearchBox'. Most functions there just for testing and need to finalize stuff soon